tests/test_helpers.py

```diff
@@ -473,62 +473,35 @@
-     "     objective: lead presents background reading and initial project plan\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - do background reading\n"
-     "       - understand math\n"
-     "4. deliverable    due date: 2020-06-01    status: converged\n"
-     "     audience: ['beginning grad in chemistry']\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - deliverable note\n"
-     "       - do this\n"
-     "       - do that\n"
-     "5. planning meeting    due date: 2020-06-01    status: converged\n"
-     "     audience: ['lead', 'pi', 'group_members']\n"
-     "     objective: develop a detailed plan with dates\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - do this\n"
-     "       - do that\n"
-     ),
-    (["helper", "u_milestone", "sb_firstprojectum", "--current"],
-     "Please choose from one of the following to update/add:\n"
-     "1. new milestone\n"
-     "3. Project lead presentation    due date: 2020-05-20    status: proposed\n"
-     "4. deliverable    due date: 2020-06-01    status: converged\n"
-     "5. planning meeting    due date: 2020-06-01    status: converged\n"
+    (["helper", "u_milestone", "--milestone_uuid", "milestone_uuid_sb1_", "sb_firstpro", "kosb_firstpro",
+      "--status", "converged", "--due-date", "2020-06-01", "--notes", "do this", "do that", "--type", "meeting"],
+     "The milestone uuid milestone_uuid_sb1_ in sb_firstprojectum has been updated in projecta.\n"
+     "The milestone uuid sb_firstpro in sb_firstprojectum has been updated in projecta.\n"
+     "The milestone uuid kosb_firstpro in sb_firstprojectum has been updated in projecta.\n"
+    ),
+    (["helper", "u_milestone", "--milestone_uuid", "kosb_firtpro",
+      "--status", "converged", "--due-date", "2020-06-01", "--notes", "do this", "do that", "--type", "meeting"],
+     "No ids were found that match your entry (kosb_firtpro).\n"
+     "Make sure you have entered the correct uuid or uuid fragment and rerun the helper.\n"
+     ),
+    (["helper", "u_milestone", "--milestone_uuid", "pl", "--status", "finished",
+      "--due-date", "2023-01-01", "--notes", "do this", "do that", "--type", "mergedpr"],
+     "Multiple ids match your entry (pl).\n"
+     "Try entering six or more characters of the uuid and rerun the helper.\n"
+     ),
+    (["helper", "u_milestone", "--projectum_id", "pl", "--name", "new milestone",
+      "--due_date", "2023-01-01", "--objective", "do all the things to complete this milestone",
+      "--notes", "do this", "do that", "--type", "mergedpr"],
+     "Projectum not found. Projecta with similar names: \n"
+     "pl_firstprojectum\n"
+     "pl_secondprojectum\n"
+     "pl_thirdprojectum\n"
+     "Please rerun the helper specifying the complete ID.\n"
+     "If your prum id looks correct, check that this id is in the collection "
+     "in the database that regolith is looking in (i.e., {rc.database}).\n"
```

Comment:
this should not be `{rc.database}` but should resolve to the name of the database in the test.  It is a concern that this test is passing!  In test driven development, always give more thought to your tests than your code.....

Reply:
right. the reason it was passing is that I had the entire phrase written as a part of the statement. I misunderstood your previous comment about providing database info to the user. will change the test to reflect the correct usage of `{rc.database}` information.